test/server/cards/01-Core/KingOfTheCrag.spec.js

```diff
@@ -0,0 +1,26 @@
+describe('King of the Crag', function () {
```

Comment:
If you want to fix it in this PR, I noticed that the comment in KingOfTheCrag.js that has the card’s ability doesn’t include the minus sign.